apps/condo/domains/ticket/schema/TicketChange.js

```diff
@@ -77,32 +77,36 @@ const TicketChange = new GQLListSchema('TicketChange', {
                 const orgEmployees = await find('OrganizationEmployee', {
                     organization: { id: orgId },
                     user: { id: userId },
+                    deletedAt: null,
+                    isBlocked: false,
                 })
+                let changedByEmployeeMessage
+
                 if (orgEmployees.length) {
-                    const existingEmployee = orgEmployees.filter(employee => !employee.deletedAt && !employee.isBlocked)
-                    if (existingEmployee.length) {
-                        const roleID = existingEmployee[0].role
-                        const role = await getById('OrganizationEmployeeRole', roleID)
-
-                        return getTranslation(translations, role.name)
-                    } else {
-                        return getTranslation(translations, 'DeletedEmployee')
-                    }
-                } else {
+                    const roleID = orgEmployees[0].role
+                    const role = await getById('OrganizationEmployeeRole', roleID)
+
+                    changedByEmployeeMessage = getTranslation(translations, role.name)
+                }
+
+                if (!changedByEmployeeMessage) {
                     const links = await find('OrganizationLink', {
                         to: { id: orgId },
                     })
                     const relatedOrgIds = links.map(link => link.from).filter(Boolean)
-                    if (!relatedOrgIds.length) return getTranslation('DeletedEmployee')
-
                     const relatedOrgEmployees = await find('OrganizationEmployee', {
                         organization: { id_in: relatedOrgIds },
                         user: { id: userId },
+                        deletedAt: null,
+                        isBlocked: false,
                     })
 
-                    const translationKey = relatedOrgEmployees.length ? 'ContactCenterEmployee' : 'DeletedEmployee'
-                    return getTranslation(translations, translationKey)
+                    if (relatedOrgEmployees.length) {
+                        changedByEmployeeMessage = getTranslation(translations, 'ContactCenterEmployee')
```

Comment:
it looks strange

Reply:
Removed 'ContactCenterEmployee' translation, used `role.name` instead